src/packages/database/database_manager.py

```diff
@@ -174,6 +174,7 @@ def __init__(self,
             connection.set_client_encoding('utf-8')
             # Create cursor, and execute the query.
             with connection.cursor() as cursor:
+                logger.debug("executing query {}", query)
```

Comment:
this has security concerns, if you're exposing the entire query.

Reply:
placed behind a `__debug__` check.